src/design-tokens/colors.yml

```diff
@@ -0,0 +1,171 @@
+aliases:
+  blue-lighter: "#8abfff"
+  blue-light: "#3d84F5"
+  blue: "#215cca"
+  blue-dark: "#183889"
+  blue-darker: "#0E1C43"
+  green-lighter: "#71f4ae"
+  green-light: "#2cdd90"
+  green: "#158466"
+  green-dark: "#105c52"
+  green-darker: "#0c3538"
+  purple-lighter: "#e98efa"
+  purple-light: "#d250f7"
+  purple: "#950cde"
+  purple-dark: "#560a9b"
+  purple-darker: "#2a085e"
+  fuchsia-lighter: "#ff80ca"
+  fuchsia-light: "#f240ae"
+  fuchsia: "#d9118f"
+  fuchsia-dark: "#950f7d"
+  fuchsia-darker: "#5c0a5c"
+  orange-lighter: "#ffad5c"
+  orange-light: "#fa8f47"
+  orange: "#f27041"
+  orange-dark: "#aa3f24"
+  orange-darker: "#622018"
+  gray-lighter: "#dee7ec"
+  gray-light: "#a5b9c6"
+  gray: "#738da0"
+  gray-dark: "#485968"
+  gray-darker: "#252b32"
+  black-transparent-85: "rgba(0, 0, 0, 0.85)"
+  white-transparent-85: "rgba(255, 255, 255, 0.85)"
+  white: "#fff"
+props:
+  primary_brand_lighter:
+    value: "{!blue-lighter}"
+    type: color
+    category: brand
+  primary_brand_light:
+    value: "{!blue-light}"
+    type: color
+    category: brand
+  primary_brand:
+    value: "{!blue}"
+    type: color
+    category: brand
+  primary_brand_dark:
+    value: "{!blue-dark}"
+    type: color
+    category: brand
+  primary_brand_darker:
+    value: "{!blue-darker}"
+    type: color
+    category: brand
+  green_lighter:
+    value: "{!green-lighter}"
```

Comment:
Do you know if you can use token values as aliases? It feels a little funny having these defined twice, didn't know if that's a limitation of Theo or not.

Reply:
I'm not sure, maybe we don't need the aliases? We could replace the value directly with hex or rgba for each color instead of duplicating code.

Reply:
I guess what I'm curious about is if this still works:

```
props:
  blue:
    value: "#XXXXXX"
    type: color
    category: example
  primary:
    value: "{!blue}"
    type: color
    category: example
```

If it does _not_, then the duplication is probably a good idea. If it _does_, then it probably isn't necessary.

Reply:
I tested this out and it looks like the aliases are necessary.